plugins/kn-admin/hack/build.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+set -o pipefail
```

Comment:
Is there a way to handle running tests?

Reply:
Yes, as we have no UT so far and will add test later.